harvester/records/formats/gbl1.py

```diff
@@ -42,7 +42,7 @@ def _dct_title_s(self) -> str | None:
         return self.parsed_data["dc_title_s"]
 
     def _gbl_resourceClass_sm(self) -> list[str]:
-        if value := self.parsed_data.get("dc_type_s"):  # noqa: SIM102s
```

Comment:
@ehanson8 - I'm betting this was my old keyboard!  When I would try and `ctrl + s` to save the file, it was often just adding an `s`.  Thankfully, those keyboard related typos are behind me.  Nice catch.